flake.nix

```diff
@@ -134,14 +137,14 @@
             ];
           };
 
+          webui = pkgs.callPackage ./webui { };
```

Comment:
Can we create a directory called `packages` where we have our packages? Like in my nix-config.